go/mysql/replication/mysql56_gtid_set.go

```diff
@@ -467,6 +467,37 @@ func (set Mysql56GTIDSet) SIDBlock() []byte {
 	return buf.Bytes()
 }
 
+// ErrantGTIDsOnReplica gets the errant GTIDs on the replica by comparing against the primary position and UUID.
+func ErrantGTIDsOnReplica(replicaPosition Position, primaryPositionStr string) (string, error) {
```

Comment:
It would be more symmetric to pass in the primaryPosition as a `Position` instead of a string.

Reply:
Fixed